source/mir/combinatorics/package.d

```diff
@@ -265,39 +264,6 @@ if (isInputRange!Range && hasLength!Range)
     return permutations(r.length).indexedRoR(r);
 }
 
-///
-pure @safe nothrow unittest
-{
-    import std.algorithm.comparison : equal;
-    import std.range : iota;
-
-    auto expectedRes = [[0, 1, 2],
-         [1, 0, 2],
-         [2, 0, 1],
-         [0, 2, 1],
-         [1, 2, 0],
-         [2, 1, 0]];
-
-    auto r = iota(3);
-    auto rp = permutations(r.length).indexedRoR(r);
-    assert(rp.equal!equal(expectedRes));
-
-    // direct style
-    auto rp2 = iota(3).permutations;
-    assert(rp2.equal!equal(expectedRes));
-}
-
-pure @safe nothrow unittest
-{
-    // is copyable?
-    import std.algorithm: equal;
-    import std.range: iota, dropOne;
-    auto a = iota(2).permutations;
-    assert(a.front.equal([0, 1]));
-    assert(a.save.dropOne.front.equal([1, 0]));
-    assert(a.front.equal([0, 1]));
-}
-
 /// ditto
```

Comment:
was this note for `Permutations` - imho `makePermutations` should be bundled with `permutations`